packages/js/product-editor/src/components/product-section-layout/product-field-section.tsx

```diff
@@ -29,11 +30,13 @@ export const ProductFieldSection: React.FC< ProductFieldSectionProps > = ( {
 		description={ description }
 		className={ className }
 	>
-		<Card>
-			<CardBody>
-				{ children }
-				<WooProductFieldItem.Slot section={ id } />
-			</CardBody>
-		</Card>
+		<>
+			<Card>
+				<CardBody>
+					{ children }
+					<WooProductFieldItem.Slot name="" section={ id } />
+				</CardBody>
+			</Card>
+		</>
```

Comment:
With the update in `ProductSectionLayout` the use of `Fragment` isn't necessary anymore, we can remove that.